share/spice/reddit_user_search/reddit_user_search.js

```diff
@@ -0,0 +1,84 @@
+(function (env) {
+    "use strict";
+    
+     env.ddg_spice_reddit_user_search = function (api_result) {
+        // Check if we have data to work with.
+        if(api_result.error) {
+            return Spice.failed('reddit_user_search');
+        }
+        Spice.add({
+            id: "reddit_user_search",
+            name: "Social",
+            data: api_result.data,
+            meta: {
+                itemType: api_result.data.name + " (User)",
+                sourceUrl: 'http://www.reddit.com/u/' + api_result.data.name,
+                sourceName: 'Reddit'
+            },
+            templates: {
+		group: 'base',
```

Comment:
I would suggest you try using the `text` or `info` templates. The `base` template is really meant as a last resort. Using a template should simplify the code a bit and will make everything look a little nicer.